deps/media-playback/media-playback/decode.c

```diff
@@ -151,7 +151,29 @@ static uint16_t get_max_luminance(const AVStream *stream)
 	return max_luminance;
 }
 
-bool mp_decode_init(mp_media_t *m, enum AVMediaType type, bool hw)
+static int get_stream(mp_media_t *m, struct mp_decode *d, enum AVMediaType type,
+		      int track)
+{
+	if (d->audio) {
```

Comment:
Nitpick: Make this function media-agnostic to allow it to be used for selecting the current video track too